print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
```

Comment:
Consider adding comments on these variables, what sort of thing you expect a value to be? What sort of thing you expect next to be?